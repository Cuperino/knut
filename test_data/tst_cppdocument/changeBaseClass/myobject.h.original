#pragma once

#include <string>

#include <CPropertyPage.h>
#include <CPropertyPage>

class CPropertyPage;

class MyObject : public CPropertyPage {
public:
    MyObject(const std::string& message);
    MyObject(const QString &msg);
    ~MyObject();
    void sayMessage();
    void sayMessage(const std::string& test);

private:
    std::string m_message;
};
