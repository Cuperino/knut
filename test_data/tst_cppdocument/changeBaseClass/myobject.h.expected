#pragma once

#include <string>

#include <KPropertyPage.h>
#include <KPropertyPage>

class KPropertyPage;

class MyObject : public KPropertyPage {
public:
    MyObject(const std::string& message);
    MyObject(const QString &msg);
    ~MyObject();
    void sayMessage();
    void sayMessage(const std::string& test);

private:
    std::string m_message;
};
